src/Tools/dotnet-monitor/Startup.cs

```diff
@@ -155,8 +155,9 @@ public void ConfigureServices(IServiceCollection services)
                     {
                         address = BindingAddress.Parse(url);
                     }
-                    catch (Exception)
+                    catch (FormatException ex)
                     {
+                        logger.LogInformation(ex.Message);
```

Comment:
Warning level. Also, add the LoggerExtensions instead of calling ILogger directly. And pass the exception through the exception parameter.